include/mod_event_pusher_rabbit.hrl

```diff
@@ -0,0 +1,14 @@
+%%%===================================================================
+%%% Metrics
+%%%===================================================================
+
+-define(METRIC_MODULE, mod_event_pusher_rabbit).
```

Comment:
Is the `METRIC_MODULE` name a standard one? `BACKEND` reads better in that context.